godel_msgs/srv/KeyenceProcessExecution.srv

```diff
@@ -0,0 +1,18 @@
+# Joint trajectory
+trajectory_msgs/JointTrajectory trajectory_approach
+trajectory_msgs/JointTrajectory trajectory_process
+trajectory_msgs/JointTrajectory trajectory_depart
+
+# wait for execution to finish before returning
+bool wait_for_execution
+
+# Use the simulated robot or the actual robot
+bool simulate
+
+# Parameters associated with the Keyence Process
+godel_msgs/ScanPlanParameters params
+
+---
+
+# Return code
+int32 code
```

Comment:
I agree in that the name of services should be indicative of the action being taken rather that a particular stage in a process; the services in the [moveit_msgs](http://wiki.ros.org/moveit_msgs) package  are name in such way.  However I realized that renaming all the services files would be a rather involved change as they are referenced in code, CMakeList, package.xml, etc.  Could we address this in a separate PR?